### PR TITLE
revert manifest to v2 and bump extension version

### PR DESCRIPTION
**Pull request by wirednkod**

### CODE REVIEW

File: projects/extension/public/manifest.json [hunk 1]
```diff
@@ -32,7 +31,6 @@
     "48": "./icons/icon-48.png",
     "128": "./icons/icon-128.png"
   },
-  "content_security_policy": {
-    "extension_pages": "script-src 'self' 'wasm-unsafe-eval'; object-src 'self'"
-  }
+  "content_security_policy": "script-src 'self' 'wasm-unsafe-eval'; object-src 'self'",
+  "web_accessible_resources": ["page.js"]
```
Comment (tomaka) [inline projects/extension/public/manifest.json:35]: There's no `page.js`

File: projects/extension/public/manifest.json [hunk 2]
```diff
@@ -4,14 +4,13 @@
...
   "short_name": "substrate-connect",
-  "version": "0.2.0",
-  "manifest_version": 3,
+  "version": "0.2.1",
+  "manifest_version": 2,
   "permissions": ["notifications", "storage", "tabs", "alarms"],
   "background": {
-    "service_worker": "background.js",
-    "type": "module"
+    "scripts": ["background.js"]
```
Comment (tomaka) [inline projects/extension/public/manifest.json:11]: As I've mentioned, you can leave the `service_worker` if it works

File: projects/extension/CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,12 @@
 
 ## [Unreleased]
 
+## 0.2.1 - 2022-10-18
+
+### Changed
+
+- Revert the extension to manifest v2 format, for Mozilla support. ([#1313](https://github.com/paritytech/substrate-connect/pull/1313))
```
Comment (tomaka) [inline projects/extension/CHANGELOG.md:9]:
```suggestion
- Revert the extension to manifest v2 format, for compatibility with Firefox. ([#1313](https://github.com/paritytech/substrate-connect/pull/1313))
```

The navigator isn't named Mozilla

File: projects/extension/public/manifest.json (hunk 2 above)
Comment (wirednkod) [reply projects/extension/public/manifest.json:11]: It is not working with `service_worker` and manifest v2

**Review by tomaka — APPROVED** [review]

**Pull request merged**